mmdet/core/post_processing/merge_augs.py

```diff
@@ -19,11 +23,40 @@ def merge_aug_proposals(aug_proposals, img_metas, rpn_test_cfg):
             For details on the values of these keys see
             `mmdet/datasets/pipelines/formatting.py:Collect`.
 
-        rpn_test_cfg (dict): rpn test config.
+        cfg (dict): rpn test config.
 
     Returns:
         Tensor: shape (n, 4), proposals corresponding to original image scale.
     """
+
+    cfg = copy.deepcopy(cfg)
+
+    # deprecate arguments warning
+    if 'nms' not in cfg or 'max_num' in cfg or 'nms_thr' in cfg:
+        warnings.warn(
+            'In rpn_proposal or test_cfg, '
+            'nms_thr has been moved to a dict named nms as'
+            'iou_threshold, max_num has been renamed as max_per_img, '
+            'name of original arguments and the way to specify '
+            'iou_threshold of NMS will be deprecated.')
+    if 'nms' not in cfg:
+        cfg.nms = Config(dict(type='nms', iou_threshold=cfg.nms_thr))
```

Comment:
Which is more appropriate, `Config` or `ConfigDict`?

Reply:
sure, `ConfigDict` would be better.